nupic/__init__.py

```diff
@@ -19,3 +19,7 @@
 # http://numenta.org/licenses/
 # ----------------------------------------------------------------------
 __version__ = "0.3.0.dev0"
+
+import os
+NUPIC_ROOT = os.environ.get('NUPIC', os.path.dirname(os.path.realpath(__file__)))
```

Comment:
I'm not sure whether this is clear, but that's what it does right now. `os.environ.get('NUPIC', 'foo')` will return either the value of `$NUPIC` (if it's set) or `'foo'`. See https://docs.python.org/2/library/stdtypes.html#dict.get for reference.